LESSON-06/katas/solution/personalLibrary.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+interface humorousBook{
+    public function containsHumor();
+}
+
+class Book{
```

Comment:
Recommendation, copy the 

task above the code, this way you will stick to what is asked, in the case of book, you are missing: 
It will also have methods open(), close(), hasRelatedInformation().